### PR TITLE
txr: update to 256.

### PR DESCRIPTION
**Pull request by natrys**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/txr/template [hunk 1]
```diff
@@ -13,7 +13,7 @@ license="BSD-2-Clause"
 homepage="http://www.nongnu.org/txr/"
 changelog="http://www.kylheku.com/cgit/txr/tree/RELNOTES?id=${pkgname}-${version}"
```
Comment (ericonr) [inline srcpkgs/txr/template:14]:
```suggestion
changelog="http://www.kylheku.com/cgit/txr/tree/RELNOTES"
```

File: srcpkgs/txr/template [hunk 2]
```diff
@@ -26,11 +26,6 @@ pre_configure() {
 	fi
 }
 
-post_configure() {
-	vsed -i config.make -e "\#^PLATFORM_CFLAGS#s#:=.*\$#:= ${CFLAGS}#"
-	vsed -i config.make -e "\#^PLATFORM_LDFLAGS#s#:=.*\$#:= ${LDFLAGS}#"
-}
-
```
Comment (ericonr) [inline srcpkgs/txr/template:33]: I think this is still relevant... Why did you remove it?

Comment (natrys) [reply srcpkgs/txr/template:33]: I think [upstream makefile](http://www.kylheku.com/cgit/txr/tree/Makefile#n34) respects the flags set by the build style now, I believe I put this long ago when it didn't use to.
